src/chrome/content/rules/Phillips_Academy_Andover.xml

```diff
@@ -17,13 +6,32 @@
 	<target host="colwizlive.andover.edu" />
 	<target host="sso.andover.edu" />
 	<target host="studentreports.andover.edu" />
-	<!--target host="mail.andover.edu" /-->
 	<target host="mypassword.andover.edu" />
 	<target host="student.andover.edu" />
 	<target host="patechmasters.com" />
 	<target host="*.patechmasters.com" />
 	
+	
+	<!-- andover.edu does not respond, but should be redirected to https://www.andover.edu to prevent MitM. -->
+	<target host="andover.edu" />
+	
+	<!-- mail.andover.edu returns an error, but should be included to protect mail.andover.edu/owa -->
+	<target host="mail.andover.edu" />
+	
+	<test url="http://andover.edu" />
```

Comment:
@jsha I don't understand, but the [Travis-CI build](https://travis-ci.org/EFForg/https-everywhere/builds/69226922) asked for test URLs:

    ERROR src/chrome/content/rules/Phillips_Academy_Andover.xml: Not enough tests (1 vs 2) for <Rule from '^http://andover\.edu/*' to 'https://www.andover.edu/'>
    ERROR src/chrome/content/rules/Phillips_Academy_Andover.xml: Not enough tests (1 vs 2) for <Rule from '^http://mail\.andover\.edu/*$' to 'https://mail.andover.edu/owa'>
    Ruleset test coverage was insufficient.